evalcache.plugin.zsh

```diff
@@ -7,7 +7,17 @@
 export ZSH_EVALCACHE_DIR=${ZSH_EVALCACHE_DIR:-"$HOME/.zsh-evalcache"}
 
 function _evalcache () {
-  local cacheFile="$ZSH_EVALCACHE_DIR/init-${1##*/}.sh"
+  local cmdHash="nohash"
+
+  # Use md5 on MacOS
+  if command -v md5 > /dev/null; then
```

Comment:
I believe `if builtin command`... will be a bit more robust from the research I just did (should make sure shell builtin gets used instead of an alias or binary).